src/cart/crt_swim.c

```diff
@@ -288,18 +288,25 @@ static int crt_swim_set_member_state(struct swim_context *ctx,
 
 	switch (state->sms_status) {
 	case SWIM_MEMBER_ALIVE:
+		/* walk the global list to execute the user callbacks */
```

Comment:
seems we should just have internal static helper function for this code block that takes in 'from', 'message' and executes the event